tests/basic.rs

```diff
@@ -35,6 +35,7 @@ fn iter_threaded() {
     assert_eq!(rx.iter().sum::<u32>(), (0..1000).sum());
 }
 
+#[cfg_attr(any(target_os = "macos", windows), ignore)]
```

Comment:
What's up with this test?

Reply:
These timeout tests are falling on mac and windows.
I tried to use release mod for these tests, but somehow I cannot make it in bound.
https://github.com/zesterer/flume/pull/17/checks?check_run_id=1076914017#step:8:415

Reply:
Hmm. I guess just terrible scheduling? Have you tried increasing the timeout constraint?

Reply:
I could, but here the default timeout is 350ms +/- 5ms, and the win32 result is 390ms,
It is a rather big measurement error. So unless we want to increase the measurement error
for windows and mac, the tests could still fall.

Reply:
Hmmm. It's very strange that the error is so large.

Reply:
Well, I'll accept the PR for now since it's not coming with any disadvantages at the moment, but I should look into exactly why this happens (presumably just very poor scheduling due to the sheer number of tests?)

Reply:
Could reduce test-threads maybe? Especially given that a lot of flume tests spawn threads themselves

Reply:
Cool. Let me do some git cleanup things, after that you could review again.

Reply:
Any chance you could cfg-disable the failing tests on the platforms they fail on?

Reply:
Done.